src/app/modules/main/profile_section/keycard/module.nim

```diff
@@ -23,6 +23,8 @@ export io_interface
 logScope:
   topics = "profile-section-profile-module"
 
+const DEFAULT_COLORID_FOR_UNDEFINED_WALLET_ACCOUNT = "undefined"
```

Comment:
This should be defined in `keypair_account_item.nim` and exposed from there and set as default value there. Also is it `""` or `"undefined"` is irrelevant, cause in that case color is not used on UI side.

Reply:
Ive kept it as undefined as I handle that value in the UI. check the Utils.qml file. But agreed will move it to a common place as you suggested